content/articles/2019-04-testthat-2-1-0.Rmarkdown

```diff
@@ -0,0 +1,112 @@
+---
+title: testthat 2.1.0
+author: Hadley Wickham
+date: '2019-04-19'
+slug: testthat-2-1-0
+description: Highlights include that `context()` is now optional, and 
+  new `expect_invisible()` and `expect_mapequal()` expectations.
+categories:
+  - package
+tags: [package, r-lib]
+photo:
+  url: https://www.pexels.com/photo/38070/
+  author: Skitterphoto
+---
+
+```{r include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+<html>
+<style>
+h2 code {
+    font-size: 1em;
+}
+</style>
+</html>
+
+We're happy to announce that [testthat 2.1.0](http://testthat.r-lib.org) is now available on CRAN! testthat makes it easy to turn your existing informal tests into formal automated tests that you can rerun quickly and easily. testthat is the most popular unit testing package for R, and is used by over 4,000 CRAN and Bioconductor packages. You can learn more about unit testing at <https://r-pkgs.org/tests.html>. 
+
+Install the latest version of testthat with:
+
+```{r, eval = FALSE}
+install.packages("testthat")
+```
+
+testthat 2.1.0 mostly contains a large number of minor improvements and bug fixes, as described in the [release notes](https://github.com/r-lib/testthat/releases/tag/v2.1.0). This blog post discusses the bigger improvements:
+
+* `context()`s are now optional.
+
+* Two new expectations: `expect_invisible()` and `expect_mapequal()`.
```

Comment:
Technically there's a list-agreement mismatch here (since the first is a sentence, and the second is a frag), but, since this matches headers, it's your call.

Reply:
Yeah, I can't think of a better way to word it, and it's not too awful, so it stays